core/src/main/java/org/mobilitydata/gtfsvalidator/parsing/RowParser.java

```diff
@@ -464,6 +469,21 @@ private String asValidatedString(
     return s;
   }
 
+  /**
```

Comment:
To simplify things, I think you could get rid of this method and just use `return asString(columnIndex, required);` where this method is used - I believe they are equivalent, as the `if` statement below is redundant.